src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/InitializeCommandLineOptions.cs

```diff
@@ -14,6 +14,9 @@ internal class InitializeCommandLineOptions : VmrSyncCommandLineOptions
 {
     [Option('r', "recursive", Required = false, HelpText = $"Process also dependencies (from {VersionFiles.VersionDetailsXml}) recursively.")]
     public bool Recursive { get; set; } = false;
-    
+
+    [Option("source-mappings", Required = true, HelpText = $"A path to the source-mappings.json file to be used for syncing.")]
```

Comment:
Either remove `$` or even better, use a constant for "source-mappings.json", I think we have it in `VmrInfo`

Reply:
copy-pasting mistake